docs/querying/api.md

```diff
@@ -892,6 +892,10 @@ The following endpoint returns various cardinality statistics about the Promethe
 ```
 GET /api/v1/status/tsdb
 ```
+- **numSeries**: This provides the number of series in the tsdb.
```

Comment:
Since all these stats are not for the full TSDB, but just for the head block, we should clarify that in the docs here.

I'm also thinking we should make it clear in the field names themselves. Either wrap them all in a `headStats: {...}` object or prefix them all with `head`?